redis/src/main/scala/zio/redis/RedisNodeExecutorLive.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 John A. De Goes and the ZIO contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.redis
+
+import zio._
+import zio.redis.RedisNodeExecutorLive._
+
+final class RedisNodeExecutorLive(
```

Comment:
Let's rename all of them:
- `RedisClusterExecutorLive` to `ClusterExecutor`
- `RedisNodeExecutorLive` to `SingleNodeExecutor`
- `RedisTestExecutorLive` to `TestExecutor`

To be completely "ZIO2 friendly" we should make them publicly accessible and move all of the "live" layers into their companion objects.

Reply:
I hope, that i did everything correct